src/cmd/login/login.go

```diff
@@ -68,25 +69,59 @@ func login(_ *cobra.Command, _ []string) error {
 			return err
 		}
 		prints.PrintCliStderr("User registered as Otterize user with user ID: %s", user.Id)
-		userId = user.Id
+		meResponse, err = c.MeQueryWithResponse(registerCtxTimeout)
+		if err != nil {
+			return err
+		}
 	} else if err != nil {
 		return err
-	} else {
-		userId = meResponse.JSON200.User.Id
 	}
 
-	// query user to get full user info
-	userResponse, err := c.UserQueryWithResponse(registerCtxTimeout, userId)
-	if err != nil {
-		return err
-	}
-
-	user := lo.FromPtr(userResponse.JSON200)
+	// FIXME: create task to
```

Comment:
to what?

Reply:
Started writing something down so I don't forget... and didn't finish. I forgot 🤦